inference-engine/thirdparty/clDNN/src/graph_optimizer/prepare_primitive_fusing.cpp

```diff
@@ -408,6 +418,8 @@ void prepare_primitive_fusing::fuse_simple_primitives(program_impl &p) {
 
             should_fuse |= input_data.is_type<space_to_batch>();
 
+            should_fuse |= input_data.is_type<reduce>();
```

Comment:
activation op can also have per-channel parameters. As I understand, you need to have an extra check here to disable fusing in such cases.